airflow/contrib/operators/gcp_bigtable_operator.py

```diff
@@ -0,0 +1,376 @@
+    https://googleapis.github.io/google-cloud-python/latest/bigtable/instance.html#google.cloud.bigtable.instance.Instance.delete
+
+    :type project_id: str
+    :param project_id: The Google Developers Console [project ID or project number]
+    :type instance_id: str
+    :param instance_id: The ID of the resource to delete, in this case instance_id
+    """
+    REQUIRED_ATTRIBUTES = ('project_id', 'instance_id')
+    template_fields = ['project_id', 'instance_id']
+
+    @apply_defaults
+    def __init__(self,
+                 project_id,
+                 instance_id,
+                 *args, **kwargs):
+        self.project_id = project_id
+        self.instance_id = instance_id
+        self._validate_inputs()
+        self.hook = BigTableHook()
+        super(BigTableInstanceDeleteOperator, self).__init__(*args, **kwargs)
+
+    def execute(self, context):
+        try:
+            self.hook.delete_instance(self.project_id, self.instance_id)
+        except google.api_core.exceptions.NotFound:
+            self.log.info("The Instance '{}' does not exist in project '{}'.".format(self.instance_id, self.project_id))
+        except google.api_core.exceptions.GoogleAPICallError as e:
+            self.log.error('An error occurred. Exiting.')
+            raise e
+
+
+class BigTableTableCreateOperator(BaseOperator, BigTableValidationMixin):
+    """
+    Creates the table in BigTable instance.
+
+    For more details about creating table have a look at the reference:
+    https://googleapis.github.io/google-cloud-python/latest/bigtable/table.html#google.cloud.bigtable.table.Table.create
+
+    :type project_id: str
+    :param project_id: The Google Developers Console [project ID or project number]
+    :type instance_id: str
+    :param instance_id: The ID of the instance that will hold the new Table
+    :type table_id: str
+    :param table_id: The ID of the Table to be created
+    """
+    REQUIRED_ATTRIBUTES = ('project_id', 'instance_id', 'table_id')
+    template_fields = ['project_id', 'instance_id', 'table_id']
+
+    @apply_defaults
+    def __init__(self,
+                 project_id,
+                 instance_id,
+                 table_id,
+                 initial_split_keys=None,
+                 column_families=None,
+                 *args, **kwargs):
+        self.project_id = project_id
+        self.instance_id = instance_id
+        self.table_id = table_id
+        self.initial_split_keys = initial_split_keys or list()
+        self.column_families = column_families or dict()
+        self._validate_inputs()
+        self.hook = BigTableHook()
+        self.instance = None
+        super(BigTableTableCreateOperator, self).__init__(*args, **kwargs)
+
+    def _compare_column_families(self):
+        table_column_families = self.hook.get_column_families_for_table(self.instance, self.table_id)
+        if table_column_families.keys() != self.column_families.keys():
```

Comment:
Are you sure those keys() list are always the same? Will they always be in the same order ? I think better (guaranteed) way is to convert the lists to set and compare them then.

Reply:
Fair point, there is no guarantee on the order here. Changed to sets